addons/OPT/dynmarkerverwaltung/Texturen.sqf

```diff
@@ -0,0 +1,19 @@
+//Texturverweise
```

Comment:
Für Dateien, die reine Preprocessor-Anweisungen enthalten (zB. defines), bietet es sich an eine andere Dateiendung, als ` .sqf`  zu nehmen. Dann sieht man auf den ersten Blick, was Skript ist und was "nur" include-Dateien sind.

In der Arma-Scripting-Community hat sich für solche Dateien die Endung `.hpp` eingenbürgert.